docs/notebooks/plot_comparisons.ipynb

```diff
@@ -0,0 +1,1958 @@
+{
```

Comment:
<p>Mmm. I see your point, but I do explain it in point (2) in the paragraph above. Perhaps I should explain better / more? The <code>camper</code> and <code>child</code>  covariates are no longer there since we averaged by <code>persons</code>.  </p><p> </p>

Reply:
Yeah, when looking at the plot it is "abstracted away". I hope you feel better soon.